src/plugins/intel_cpu/src/exec_network.cpp

```diff
@@ -126,6 +128,18 @@ ExecNetwork::ExecNetwork(const InferenceEngine::CNNNetwork &network,
         do {
             for (auto&& task : tasks) {
                 task = [this] {
+                    // SSE runtime check is needed for some ATOM machine, which is x86-64 but w/o SSE
+                    static Xbyak::util::Cpu cpu;
+                    if (cpu.has(Xbyak::util::Cpu::tSSE)) {
+                        if (this->_cfg.denormalsOptMode == Config::DenormalsOptMode::DO_On) {
```

Comment:
@xipingyan This function is used in scenarios where the application and CPU inference are run in different threads. So SEQ should not support it. 

Then Please add different default setting of _cfg.denormalsOptMode for TBB/OMP based on macro definition `OV_THREAD_TBB`, `OV_THREAD_OMP` and `OV_THREAD_TBB_AUTO`. 

And please add test case in ov_cpu_unit_tests to cover this logic.

Reply:
hi @wangleis @dmitry-gorokhov ,
When I added unit tests for it, I found a new problem.
If the customer calls `start_async`, current solution works fine, but if the customer calls` infer`, it will not work because inference runs in the customer's main thread when `infer` is called.

Reply:
@xipingyan The behavior you mentioned is right. 

If this function need to cover `infer()`, I propose to keep the function in main threads as well. And just skip the function in main threads and enable it in sub threads by default. Then `start_async()` will enable this function by default and `infer()` can enable this function by user setting. 

Please document this different behavior between `start_async()` and `infer()`.

Reply:
Updated and added unit test. Please help review it again.

Reply:
Test had been added cpu functional test.